client/my-sites/themes/controller.jsx

```diff
@@ -99,20 +97,24 @@ export function fetchThemeFilters( context, next ) {
 		return next();
 	}
 
-	const unsubscribe = store.subscribe( () => {
-		const fetchErr = getThemeFiltersRequestError( store.getState() );
-		if ( fetchErr ) {
-			debug( `Theme fetch error: ${ JSON.stringify( fetchErr ) }` );
-			unsubscribe();
-			return next( new Error( 'Error fetching theme filters.' ) );
-		}
-
-		if ( Object.keys( getThemeFilters( store.getState() ) ).length > 0 ) {
-			unsubscribe();
-			return next();
-		}
+	const filtersRequest = wpcom.req.get( {
+		path: '/theme-filters',
+		apiVersion: '1.2',
+		query: context.lang ? { locale: context.lang } : {},
```

Comment:
When we start using the second argument of `wpcom.req.get()` perhaps it makes more sense to pass that `locale` there?

Reply:
done